src/downloader/chain_config.rs

```diff
@@ -0,0 +1,21 @@
+use anyhow;
+use serde::Deserialize;
+use std::collections::HashMap;
+use toml;
+
+pub struct ChainsConfig(pub HashMap<String, ChainConfig>);
+
+#[derive(Hash, Deserialize)]
+pub struct ChainConfig {
+    pub id: u32,
+    pub genesis: String,
+    pub fork_blocks: Vec<u64>,
```

Comment:
`BTreeSet<u64>`

Reply:
I've used Vec is for 2 reasons:

1. I'd like this structure to represent the TOML as it is without any smartness. It is an array in TOML, so I've used Vec.
2. The generated `Forks` type from the .proto file actually accepts Vec.

If we plan to use it in some logic as a Set, we could change it later, or potentially could be in a different type (mapped from this one). I can make a note about this. Do we have a trello board or something?

Reply:
It's OK to have "smartness" here - we expect a _set_ of forks in user-provided config after all 🙂 

Also, sentry's fork id may get calculated incorrectly if the forks are provided in the wrong order, or duplicated.

Reply:
ok, I understand now, this would act as a protection from bad configs